<!DOCTYPE html>.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Cooking up a Storm</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+      <header class="header-background">
+        <h1>Cooking up a Storm</h1>
+        <nav class = "navibar">
+            <a href= "<!DOCTYPE html>.html">HOME</a>
+            <a href = "aboutme.html"> ABOUT ME</a>
+        </nav>
+      </header>
+      <main>
+          <h1>Five Star Breakfast</h1>
+          <h2>
+          <a href = "french.html"> French Toast<a>
+            <div class="search-container">
+                <img class = "search-item" src = "https://mccormick.widen.net/content/z8hiapwdst/original/quick_and_easy_french_toast_new_800x800.webp">
+            </div>
+            </h2>
+          <h2><a href = "pancake.html"> Pancakes<a>
+            <div class="search-container">
```

Comment:
I noticed that you use class here and a few other places, but you don't make use of your classes in your styles anywhere